### PR TITLE
Use subtler UI sounds

### LINKED ISSUE #858: Need Option to disable sound on Mac
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
0.12.0 (0.96.3)
**What operating system and version?**
macOS Sierra 
**What did you do?**
Use the app
**What did you expect to happen?**
Have an option to disable sound
**What actually happened?**
Dont see an option to disable sound

### PR DESCRIPTION
**Pull request by feross**
Fixes #858

### CODE REVIEW

File: src/renderer/lib/sound.js [hunk 1]
```diff
@@ -19,19 +19,19 @@ const sounds = {
   },
   DELETE: {
     url: 'file://' + path.join(config.STATIC_PATH, 'sound', 'delete.wav'),
-    volume: VOLUME
+    volume: VOLUME * 0.8
```
Comment (dcposch) [inline src/renderer/lib/sound.js]: Doesn't that mean these sounds are now louder than before? 0.4 * 0.8 > 0.15

Comment (Flet) [inline src/renderer/lib/sound.js]: The sounds themselves are also different @dcposch :)

Comment (feross) [inline src/renderer/lib/sound.js]: @dcposch You're right. The delete sound is unchanged, so it is now louder than before. Will fix.

**Comment by feross** [comment]
I just tweaked the sound levels again. Can I get a review?

**Pull request merged**